package.json

```diff
@@ -153,6 +153,36 @@
 					"title": "Draw.io Local Storage",
 					"description": "Only change this property if you know what you are doing. Manual changes to this property are not supported!"
 				},
+				"hediet.vscode-drawio.presetColors": {
+					"title": "Preset Colors",
+					"description": "Color codes for the upper palette in the color dialog (no leading # for the color codes).",
```

Comment:
Could you describe how a color code would look like? Is `red` ok? or `RRGGBBAA` for specifying alpha? Maybe this can be added as description for the item type.